api_client/python/timesketch_api_client/client.py

```diff
@@ -605,8 +605,87 @@ def list_sigma_rules(self, as_pandas=False):
             rules.append(index_obj)
         return rules
 
+    def list_sigmarules_rules(self, as_pandas=False):
+        """Fetches Sigma rules from the database.
+        Fetches all Sigma rules stored in the database on the system
+        and returns a list of SigmaRule objects of the rules.
+
+        Args:
+            as_pandas: Boolean indicating that the results will be returned
+                as a Pandas DataFrame instead of a list of SigmaRuleObjects.
+
+        Returns:
+            - List of Sigme rule object instances
+            or
+            - a pandas Dataframe with all rules if as_pandas is True.
+
+        Raises:
+            ValueError: If no rules are found.
+        """
+        rules = []
+        response = self.fetch_resource_data("sigmarule/")
+
+        if not response:
+            raise ValueError("No rules found.")
+
+        if as_pandas:
+            return pandas.DataFrame.from_records(response.get("objects"))
+
+        for rule_dict in response["objects"]:
+            if not rule_dict:
+                raise ValueError("No rules found.")
+
+            index_obj = sigma.Sigma(api=self)
+            for key, value in rule_dict.items():
+                index_obj.set_value(key, value)
+            rules.append(index_obj)
+        return rules
+
+    def get_sigmarule_rule(self, rule_uuid):
```

Comment:
same here, please rename. 

Also, can we add a logger here telling the user that they are using a deprecated method?

Reply:
done